install/download-models.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+cd ..
+
+if [ -e "bergamot-models" ]; then
+    echo "Already cloned"
+else
+    echo "Cloning https://github.com/mozilla-applied-ml/bergamot-models.git"
+    git clone https://github.com/mozilla-applied-ml/bergamot-models.git
+fi
+
+cd bergamot-models/prod
+
+echo "Extracting models"
+gzip -d -r *
```

Comment:
Nit: Ending files with newline makes diffs easier to read after file is appended in future revisions